### DISCUSSION TITLE
nuxt dev randomly stopping

### DISCUSSION THREAD

**Discussion by mael-queau** [nuxt/nuxt#17102] [Questions]
Hi, I've been learning how to use Nuxt 3 lately, since we're rewriting a project with it, and I've been facing a bit of an inconvenience. When using `nuxt dev` (I'm using Yarn as a package manager), upon saving a file (mostly using TypeScript), the process will sometimes exit without a warning, an error or a notice, just as if it were its normal cycle. I haven't been able to create a minimal reproduction repo, so I can't really open an issue about it... Have any of you encountered this before? I've tried searching the repo for related issues or discussions, but couldn't find anything.

As for a frequency estimate, I'd say this usually happens anywhere between 1 and 30 file saves.

EDIT: [Here](https://i.imgur.com/Ved8vUw.gif) is a GIF documenting what it looks like in my terminal.

**Comment by zihaoyy** [comment]
> Hi, I've been learning how to use Nuxt 3 lately, since we're rewriting a project with it, and I've been facing a bit of an inconvenience. When using `nuxt dev` (I'm using Yarn as a package manager), upon saving a file (mostly using TypeScript), the process will sometimes exit without a warning, an error or a notice, just as if it were its normal cycle. I haven't been able to create a minimal reproduction repo, so I can't really open an issue about it... Have any of you encountered this before? I've tried searching the repo for related issues or discussions, but couldn't find anything.
> 
> As for a frequency estimate, I'd say this usually happens anywhere between 1 and 30 file saves.
> 
> EDIT: [Here](https://i.imgur.com/Ved8vUw.gif) is a GIF documenting what it looks like in my terminal.

I think there are two possibilities: one is that you have enabled the auto-save function of the editor, and the other is that you are advised to delete the `.nuxt` folder and run it again

**Reply by mael-queau** [reply]
Hi, sorry for the long delay since my initial question, but we actually haven't been using `nuxt dev` since, so I didn't encounter that issue for a while. It still happens though, and since I've taken up front-end development again today at work, I was met with the exact same thing happening the exact same way. I've bumped the version to `3.0.0-rc.6` and tried deleting `.nuxt` and starting it up again, but to no avail.
